.travis.yml

```diff
@@ -26,6 +26,7 @@ install:
   - npm install -g npm
   - npm install -g poltergeist
   - bundle install
+  - ENABLE_TURBOLINKS_5=TRUE bundle install
```

Comment:
Any gems that are used in spec/dummy need to be installed before running the tests. This basically tells travis-ci to install turbolinks classic and turbolinks 5.